src/templates/BillOfLadingBimcoStandard/BillOfLadingBimcoStandard.tsx

```diff
@@ -0,0 +1,265 @@
+import React from "react";
+import { v2 } from "@govtechsg/open-attestation";
+import { TemplateProps } from "@govtechsg/decentralized-renderer-react-components";
+import "./css/bimco.css";
+
+interface Address {
+  street: string;
+  streetNumber: string;
+  floor?: string;
+  postCode: string;
+  city: string;
+  stateRegion: string;
+  country: string;
+}
+
+interface Party {
+  partyName: string;
+  taxReference: string;
+  address: Address;
+  phone: string;
+  email: string;
+  url?: string;
+}
+
+interface BimcoV1Template extends v2.OpenAttestationDocument {
+  $template: {
+    // Included $template property
+    name: string;
+    type: v2.TemplateType.EmbeddedRenderer;
+    url: string;
+  };
+  issuers: any[];
+  shipper: { party: Party };
+  consignee: { party: Party };
+  notifyParty: { party: Party };
+  freightPayableAsPerCharterPartyDated: string;
+  portOfLoading: {
+    location: {
+      locationName: string;
+      UNLocationCode: string;
+    };
+  };
+  portOfDischarge: {
+    location: {
+      locationName: string;
+      UNLocationCode: string;
+    };
+  };
+  vessel: string;
+  referenceNo: string;
+  billOfLadingNo: string;
+  descriptionOfGoods: string[];
+  cargoGrossWeight: string;
+  measurement: string;
+  dateOfIssue: string;
+  shippedOnBoardDate: string;
+  signedBy: string;
+  SCAC: string | null;
+  termsAndConditions: string;
+  cargoShippedOnDeck: string | null;
+}
+
+export const BillOfLadingBimcoStandard: React.FunctionComponent<TemplateProps<BimcoV1Template>> = ({ document }) => {
```

Comment:
Can we add a V1 to the name so when we are referencing this template we know the version of the template we are using.

Reply:
@isaackps should be added! 👍🏻